util/CMakeLists.txt

```diff
@@ -43,11 +48,17 @@ else()
     find_package(cryptopp REQUIRED)
 endif()
 
-target_link_libraries(util PUBLIC Threads::Threads ${CRYPTOPP_LIBRARIES})
-target_include_directories(util PUBLIC include ${CRYPTOPP_INCLUDE_DIRS})
+target_link_libraries(util PUBLIC
+        Threads::Threads ${CRYPTOPP_LIBRARIES}
+        stdc++fs)
+# corebft
+target_include_directories(util PUBLIC include ${CRYPTOPP_INCLUDE_DIRS} ../bftengine/include/bftengine include/openssl)
```

Comment:
util cannot include bftengine, it will create a circular dependency

Reply:
I need to use `ReplicaConfig` class in `utils.cpp`. The `ReplicaConfig` class is defined in `bftengine/include/bftengine`. What is the other way to do this?

Reply:
Changes done https://github.com/arc-vmware/concord-bft/pull/12/commits/6aaca0e61da69ebbaa897378ee75083aff98d4e4